tools/generate_standards.py

```
@@ -0,0 +1,21 @@
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parents[1]
+XML = ROOT / "docs/standards/rfc/draft-hyprstream-privacypass-pqhybrid-00.xml"
+
+
+def main() -> int:
+    command = [
+        "uvx", "--from", "xml2rfc==3.34.0", "xml2rfc",
+        "--no-network", "--text", "--html", "--path", str(XML.parent), str(XML),
+    ]
+    return subprocess.run(command, cwd=ROOT).returncode
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)